### PR TITLE
use default browser styling

### PR DESCRIPTION
**Pull request by dvnrsn**

### CODE REVIEW

File: src/rich-text-editor.component.js [hunk 1]
```diff
@@ -205,6 +205,16 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
...
+
+  function getBrowserDefaultStyling() {
+    if (props.placeholderColor) {
+      return `color: ${props.placeholderColor};`
+    } else if (!!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)) {
+      return `color: rgb(117, 117, 117);`
+    } else {
+      return `opacity: 0.54;`
+    }
+  }
```
Comment (dvnrsn) [inline src/rich-text-editor.component.js:217]: image
Seems to look good with safari (to be honest didn't check IE)

File: src/rich-text-editor.component.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   useEffect(() => {
     if (props.placeholder) {
       const styleElement = document.createElement('style')
-      styleElement.textContent = `.bandicoot-id-${bandicootId.current}:empty:before { content: attr(data-placeholder); color: ${props.placeholderColor}; }`
+      styleElement.textContent = `.bandicoot-id-${bandicootId.current}:empty:before { content: attr(data-placeholder); ${getBrowserDefaultStyling() }`
```
Comment (dvnrsn) [inline src/rich-text-editor.component.js]: sloppy, missed a `}` which it works without for some reason?

File: src/rich-text-editor.component.js (hunk 1 above)
Comment (dvnrsn) [reply src/rich-text-editor.component.js:217]: the real question is who at mozilla was like "yeah we need it to be .54, not .5"

Comment (internettrans) [reply src/rich-text-editor.component.js:217]: haha

**Review by internettrans — APPROVED** [review]

**Pull request merged**